traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/postgres.go

```diff
@@ -482,8 +505,22 @@ func validateConfig(cfg Config) error {
 		"port":                  validation.Validate(cfg.Port, validation.By(tovalidate.IsValidPortNumber)),
 		"max_connections":       validation.Validate(cfg.MaxConnections, validation.Min(0)),
 		"query_timeout_seconds": validation.Validate(cfg.QueryTimeoutSeconds, validation.Min(0)),
-		"aes_key_location":      validation.Validate(cfg.AesKeyLocation, validation.Required),
 	})
+	aesKeyLocSet := cfg.AesKeyLocation != ""
+	hashiCorpVaultSet := cfg.HashiCorpVault != nil
```

Comment:
maybe add something like this `&& *cfg.HashiCorpVault != HashiCorpVault{}` so people can leave an empty field in the cdn.conf? not important to me if you think thats not worth doing, just an idea cause i tried testing it that way

Reply:
sure, I can live with that!